test/tracing/mzcompose.py

```diff
@@ -26,12 +32,12 @@
 
 def workflow_default(c: Composition) -> None:
     """Tests the dynamic tracing setup on environmentd"""
-    c.workflow("with-otel")
-    c.workflow("without-otel")
+    c.workflow("with-everything")
```

Comment:
I'd prefer not having to list the workflows manually (assuming the order doesn't matter), easy to forget to list a new one:
```python
    for i, name in enumerate(c.workflows):
        if name != "default":
            with c.test_case(name):
                c.workflow(name)
```
Using c.test_case around the workflows also has the advantage to be able to run the remaining tests when one failed.

Reply:
til about this! will update and merge!